src/data/javascript-quiz.ts

```diff
@@ -1249,7 +1249,7 @@ const javascriptQuiz = [
     Question: "In JavaScript, what will the following code return: typeof a?",
     Answer: "undefined",
     Distractor1: "not defined",
-    Distractor2: "",
+    Distractor2: "object",
```

Comment:
Please make sure to update all of the distractors as outlined in the issue 👍

Reply:
Changes have been implemented